src/core/client/ui/components/TextField/TextField.css

```diff
@@ -1,8 +1,25 @@
 .root {
   composes: textField from "talk-ui/shared/typography.css";
-  background: var(--palette-common-white);
-  border: 1px solid var(--palette-grey-lighter);
+  position: relative;
+  display: block;
+  padding: calc(0.5 * var(--spacing-unit));
   box-sizing: border-box;
   border-radius: var(--round-corners);
-  padding: calc(0.5 * var(--spacing-unit));
 }
+
+.colorRegular {
+  backgroun-color: var(--palette-common-white);
+  color: var(--palette-common-black);
+  border: 1px solid var(--palette-grey-light);
+}
+
+.colorError {
+  background-color: var(--palette-common-white);
+  border-color: var(--palette-error-main);
+  border: 2px solid #FA4643;
+}
+
+.fullWidth {
+  width: 100%;
+}
+
```

Comment:
We should add the styling for placeholder here, see figma file: https://www.figma.com/file/NdlkTfgQNPSglYvYUoVVw6EH/Coral-Styles?node-id=60%3A434